cnd/src/storage.rs

```diff
@@ -1,22 +1,23 @@
+use crate::swap_protocols::rfc003::{DeriveIdentities, DeriveSecret};
```

Comment:
Storage is a new 'clear' data type, can we please not use any rfc003 stuff in it. If we need these traits then they should be moved out of the rfc003 module. I vaguely recollect some complexity whet I tried to do this before, I would settle for a GitHub issue and a comment in the code linking to the issue.

Reply:
This is done in https://github.com/comit-network/comit-rs/pull/2778